app/routes/courts.js

```diff
@@ -1,13 +1,49 @@
-const { getCourts } = require('../repos/courts')
+const { getCourts, createCourt, deleteCourt } = require('../repos/courts')
+const { getCallingUser } = require('../auth/get-user')
+const { createCourtSchema } = require('../schema/court/create')
 
-module.exports = {
-  method: 'GET',
-  path: '/courts',
-  handler: async (request, h) => {
-    const courts = await getCourts()
+module.exports = [
+  {
+    method: 'GET',
+    path: '/courts',
+    handler: async (request, h) => {
+      const courts = await getCourts()
```

Comment:
Do you not need to query the courts to include soft-deleted records? Can't remember if possible to re-enable a court

Reply:
I don't think so, we aiming to not show them aren't we on the dropdowns?

Reply:
The api will now handle create of old records.  It's possible a query param could be added in future to this to get soft deleted records as well, but the consensus was no list 🤷.  Reckon we can update when needed.  This ticket has already been downgraded, so trying not worth the extra work at this stage IMO.